src/components/product-icon/types.ts

```diff
@@ -8,6 +8,6 @@ import { ProductSlug } from 'types/products'
 type SvgElementProps = JSX.IntrinsicElements['svg']
 
 export interface ProductIconProps extends SvgElementProps {
-	productSlug: ProductSlug
+	productSlug: ProductSlug | 'vault-secrets' | 'vault-radar'
```

Comment:
Another option is to update the enum directly here: https://github.com/hashicorp/dev-portal/blob/main/src/types/products.ts#L9-L20

Reply:
I decided not to update the `ProductSlug` type because it is used so widely in the codebase. I think another PR would be a better place to update this and make sure it doesn't break any other pages that use this type.